web/src/common/constants.ts

```diff
@@ -0,0 +1,434 @@
+/**
```

Comment:
Since takes > 200 lines for 1 constant, I would suggest this is to be put in a standalone file (eg: timezones.ts)

Reply:
Alright sure